debian/xenial/cri-tools/debian/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+CRI_TOOLS_VERSION = {{ .Version }}
+
+build:
+	echo noop
+
+binary:
+	mkdir -p ./bin
+	curl -sSL --fail --retry 5 \
+		"{{ .DownloadLinkBase }}/crictl-v$(CRI_TOOLS_VERSION)-linux-{{ .Arch }}.tar.gz" \
```

Comment:
This is probably ok but has the consequence e.g. crictl v1.11.1 must exist before we can build debs for k8s :thinking: 
Should think about this flow a bit

Reply:
The alternatives are:
- lookup the latest patch version from the v1.Xx branch of k8s using the github api and use that
- hardcode

Reply:
:/ I don't quite follow, the crictl version is hardcoded 

https://github.com/kubernetes/release/pull/570/files#diff-c3bb0411929a62fea9b535a1818eeca5R4

which is defined

https://github.com/kubernetes/release/pull/570/files#diff-f58aa88e13f0c549e4f5ee29e60636e5R27

so it's already hardcoded?

Reply:
Oh yeah, I misread the current code wrt this. However, cri-tools is gonna adopt a versioning scheme similar to k8s: `v1.11.0` is the next version FYI (being released on Monday).
Hence I was thinking you already coupled the k8s version and cri-tools version, but you didn't.
Hardcoding this for now is fine by me